examples/worldstate/custom_logger.py

```diff
@@ -0,0 +1,25 @@
+# queries are just another way to get the data to the corresponding objects.
```

Comment:
This file should just be in your readme for an example... This doesn't really make sense to me.

Reply:
..what do you want o.O

Reply:
this in the readme